src/test/java/org/jitsi/meet/test/base/TestSettings.java

```diff
@@ -61,27 +61,11 @@ public static Properties initSettings()
             // setting system properties
             System.setProperties(p);
 
-            initFolders();
-
```

Comment:
Don't we need appium screenshots folder creation?

Reply:
it is done as part of https://github.com/jitsi/jitsi-meet-torture/blob/master/src/test/java/org/jitsi/meet/test/base/FailureListener.java#L63